server/db/models/plant.js

```diff
@@ -73,4 +73,29 @@ module.exports = db.define('plant', {
     image: {
         type: Sequelize.TEXT
     }
+},
+{
+    classMethods: {
+        getSunPlants: function() {
+            return this.findAll({
+                where: {
+                    sun: 2
+                }
+            })
+        },
+        getShadePlants: function() {
+            return this.findAll({
+                where: {
+                    sun: 0
+                }
+            })
+        },
+        getPartShadePlants: function() {
+            return this.findAll({
+                where: {
+                    sun: 1
+                }
+            })
+        }
```

Comment:
I like that you're using some model methods to keep things clear and refactorable.